modules/developer_manual/pages/app/tutorial/development_environment.adoc

```diff
@@ -50,17 +64,18 @@ In `appinfo/info.xml`, add the following XML, changing it as necessary:
     <namespace>OwnNotes</namespace>
     <category>tool</category>
     <dependencies>
-        <owncloud min-version="9" />
-        <owncloud max-version="10" />
+        <owncloud min-version="10.0.0" max-version="11.0.0.0" />
```

Comment:
max-version="10"

Reply:
Well spotted. Thanks.

Reply:
Is `<owncloud min-version="9.2.0" max-version="10.0.0" />` suitable?